broadcast/skins/broadcast/src/coffee/roca.coffee

```diff
@@ -95,7 +113,17 @@
       target.data 'roca-url', url
       target.loading().reload()
       if (el.data 'roca-trap-submit') is 'yes'
-        target.funnelSubmit()
+        options = { "onCompleteEvent": el.data 'roca-submit-complete-event' }
+        target.funnelSubmit(options)
+
+      # Optionally reload container when specific events are fired, delayed by
+      # a specified time
+      event = el.data 'roca-refresh-on'
+      if event
+        delay = el.data 'roca-refresh-delay'
```

Comment:
If we are doing something on a timer, then it's probably wrong. I suggest dropping the delay altogether, and *manually* triggering the event in the form handler as need.